src/models/PrimaryAuth.js

```diff
@@ -110,7 +110,7 @@ export default BaseLoginModel.extend({
       } else {
         primaryAuthPromise = this.doTransaction(function(transaction) {
           return this.doPrimaryAuth(authClient, signInArgs, transaction.authenticate);
-        });
+        }, true);
```

Comment:
Does true mean "yes, hide spinner"?

Reply:
take a look at `doPrimaryAuth` function..

Reply:
Found in `doTransaction`, the second arg is `rethrow`. So it _was_ swallowing the error, but passing `true` will rethrow the error and then trigger 'removeLoading' on line 126. Cool.

Reply:
Yup.. You got it.

Reply:
Nice I didn't know this either 🙂